2-TokenCache/Console-TokenCache/CacheSettings.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Microsoft.Identity.Client.Extensions.Msal;
+using System.Collections.Generic;
+
+namespace Console_TokenCache
+{
+    public static class CacheSettings
```

Comment:
Please update as follows: 

```csharp

// computing the root directory is not very simple on Linux and Mac, so a helper is provided
 private static readonly string s_cacheFilePath =
            Path.Combine(SharedUtilities.GetUserRootDirectory(), "msal.contoso.cache");

public const string CacheFileName = Path.GetFileName(s_cacheFilePath);
public const string CacheDir = Path.GetDirectoryName(s_cacheFilePath),


public const string KeyChainServiceName = "Contoso.MyProduct";
public const string KeyChainAccountName = "MSALCache";

```

Reply:
@bgavrilMS where is `SharedUtilities.GetUserRootDirectory()` ?

Reply:
Sorry, that's internal. But we expose it publicly as: 

`MsalCacheHelper.UserRootDirectory`

Reply:
Done. Could you please test it again @bgavrilMS ?

Reply:
Sorry Tiago, I don't have time to test this. It should be pretty easy for you to test. If you only changed the values here, there is little risk really, they are mostly labels.

Reply:
I dont have a MAC but anyways.

Reply:
I'm running tests with similar settings on our CI, so this should all work.